eng/pipelines/common/variables.yml

```diff
@@ -15,6 +15,10 @@ variables:
   value: ${{ and(eq(variables['System.TeamProject'], 'public'), ne(variables['Build.Reason'], 'PullRequest')) }}
 - name: isNotFullMatrix
   value: ${{ and(eq(variables['System.TeamProject'], 'public'), eq(variables['Build.Reason'], 'PullRequest')) }}
+- name: isSmokeTestBuild
+  value: ${{ and(not(in(variables['Build.DefinitionName'], 'manual-runtime-staging', 'manual-runtime')), eq(variables['System.TeamProject'], 'public'), eq(variables['Build.Reason'], 'PullRequest')) }}
```

Comment:
```
isSmokeTestbuild    = (!manual && PR)
isNotSmokeTestbuild = (manual || !PR)
```
Should we name these two to be indicative of when they would be true, like `isNotManualAndIsPR`, and `isManualOrIsPR`? What to do based on this can then be determined where they are used, for example, for `RunSmokeTestOnly=$(isNotManualPR)`.

Reply:
It would also make it easier to read conditions:

```yml
...
           eq(variables['isNotSmokeTestBuild'], true),
           eq(variables['isFullMatrix'], true))
...
```

.. vs ..

```yml
...
           eq(variables['isNotManualAndIsPr'], true),
           eq(variables['isFullMatrix'], true))
...
```

Reply:
I'm open to ideas :-)